### ISSUE TITLE
Otsar Hachayal login fails

### ISSUE THREAD

**Issue by hcosteff** [eshaham/israeli-bank-scrapers#441]
worked just a few weeks ago:
```- Start scraping from: Mar 1, 2020
- Combine installments: No
- Save to location: /budgeter/download
- Create single report: No
- Include future Transactions: Yes
- Include pending Transactions: Yes
Run task scrapers
scraping Bank Otsar Hahayal
Bank Otsar Hahayal: START_SCRAPING
Bank Otsar Hahayal: INITIALIZING
Bank Otsar Hahayal: TERMINATING
Bank Otsar Hahayal: END_SCRAPING
success: false
error type: GENERIC
error: waiting for selector "#login_btn" failed: timeout 30000ms exceeded
Error: waiting for selector "#login_btn" failed: timeout 30000ms exceeded
    at exports.default (/home/budgeter/israeli-ynab-updater/src/scrape/scrape-base.js:61:11)
    at <anonymous>
(node:29) UnhandledPromiseRejectionWarning: Error: waiting for selector "#login_btn" failed: timeout 30000ms exceeded
    at exports.default (/home/budgeter/israeli-ynab-updater/src/scrape/scrape-base.js:61:11)
    at <anonymous>
(node:29) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:29) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.```

**Comment by esakal** [comment]
@hcosteff did you resolve it or is it still happening?

**Comment by hcosteff** [comment]
Yes I resolved, thanks.
The issue was that the new login for otsar was fixed in IsraeliBankScrapers, but IsraeliYnabUpdater was not updated to use the right version of IsraeliBankScrapers.

When manually setting the dependency to a local version of IsraeliBankScrapers, it was not clear to me that I had to run `npm build` before using as a dependency.

This might be because I am coming from python/c++ and I have not worked with node a lot, but I tore my hair out for many hours before guessing i had to run "build" lol.

I suggest to add a little explanation of this in readme.
